docs/tutorials/chisel.rst

```diff
@@ -1,10 +1,8 @@
 Install slices in a rock
 ========================
 
-In this tutorial, you will create a lean rock that contains a fully functional
-OpenSSL installation, and you will verify that it is functional by loading the
-rock into Docker and using it to validate the certificates of the Ubuntu
-website.
+In this tutorial, you will create a lean hello-world rock that uses chisel
+slices, and then compare the resulting rock with the one created without slices.
```

Comment:
```suggestion
In this tutorial, you will create a lean hello-world rock that uses chisel
slices, and then compare the resulting rock with the one created without slices.
```

also deserves a link to the tutorial where the other rock (without slices) is created